src/NUnitFramework/framework/nunit.framework-netstandard13.project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
@ChrisMaddock are you building in Visual Studio 2017? I am and get a whole raft of warnings compiling master too. I think it is because 2017 uses the new MSBUILD. I could not update to 1.6.1 I think because it requires the newer 2017 tooling, but we are using the old PCL project files so we can support building in 2015. At some point, we are going to have to make the call on moving to Visual Studio 2017 because not requiring it is making it really hard to support newer platforms.

For now, I downgraded the .NET Core SDK back to what it was originally and left the projects targeting 1.6.0.